src/components/FilterTabLabel.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Text, nullable } from '@taskany/bricks';
+import { gapXs, gray10 } from '@taskany/colors';
+import styled from 'styled-components';
+
+interface TabLabelProps {
+    text: string;
+    selected?: string[];
+}
+
+const StyledLabelWrapper = styled.div`
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    gap: ${gapXs};
+    width: 100%;
+`;
+
+const StyledBreakText = styled(Text)`
+    word-break: break-all;
```

Comment:
Could u create issue in bricks and move it as prop to Text component? I think it would be much better because we have a lot of places with same kind of usage. Thanks 🙏🏼

Reply:
https://github.com/taskany-inc/bricks/issues/463